crates/topos/tests/config.rs

```diff
@@ -147,12 +147,14 @@ mod serial_integration {
     }
 
     /// Test node init env arguments
+    #[rstest]
     #[tokio::test]
-    async fn command_init_precedence_env() -> Result<(), Box<dyn std::error::Error>> {
-        let tmp_home_directory = tempdir()?;
-
+    async fn command_init_precedence_env(
+        create_folder: PathBuf,
+    ) -> Result<(), Box<dyn std::error::Error>> {
+        let tmp_home_directory = create_folder;
         // Test node init with env variables
-        let node_init_home_env = tmp_home_directory.path().to_str().unwrap();
+        let node_init_home_env = tmp_home_directory.to_str().unwrap();
```

Comment:
```suggestion
        let node_init_home_env = tmp_home_directory.to_str().expect("path names are valid utf-8");
```

(they're not, but that it the expectation here)

Reply:
If we're in here cleaning things up, can't we return an error here instead of panicking?

Reply:
This is tests, I'm ok to let the panic